src/v2/Apps/Fair/Components/FairHeader/FairHeaderImage.tsx

```diff
@@ -94,6 +110,19 @@ export const FairHeaderImageFragmentContainer = createFragmentContainer(
           _2x: cropped(width: 750, height: 1000, version: "wide") {
             src: url
           }
+          # > xs
+          md_1x: cropped(width: 600, height: 800, version: "wide") {
+            src: url
+          }
+          md_2x: cropped(width: 1200, height: 1600, version: "wide") {
+            src: url
+          }
+          lg_1x: cropped(width: 900, height: 1200, version: "wide") {
+            src: url
+          }
+          lg_2x: cropped(width: 1800, height: 2400, version: "wide") {
+            src: url
+          }
```

Comment:
I think it's high time I exposed a `srcSet` field in metaphysics, this is getting old fast.